.github/workflows/check.yml

```diff
@@ -110,6 +110,27 @@ jobs:
           path: build/target/hale-studio-*macosx*.dmg
           retention-days: 14
 
+      - name: Set up Trivy
```

Comment:
Why don't you use the trivy GitHub action as before?

Reply:
If I don't set up/install trivy  I get an error as:
```
Run trivy fs --severity CRITICAL,HIGH build/target/hale-studio-linux-trivy --format sarif --output trivy-results.sarif
/home/runner/work/_temp/1ea9265f-f612-4cca-8[7](https://github.com/halestudio/hale/actions/runs/9449564750/job/26026128237?pr=1159#step:10:8)d1-b6e3a263f8f2.sh: line 1: trivy: command not found
```
See [here](https://github.com/halestudio/hale/actions/runs/9454619259/job/26042573968?pr=1159)